src/stories/Library/event-list-item/event-list-item.scss

```diff
@@ -0,0 +1,152 @@
+.event-list-item {
+  @include show-svg-on-hover();
+
+  text-decoration: none;
+  display: grid;
+  padding: $s-lg;
+  row-gap: $s-md;
+  max-width: $layout__max-width;
+  background-color: #fcfcfc;
+  margin: 0 auto;
+}
+.event-list-item__image-container {
+  background: green; // Placeholder for image background / component
```

Comment:
Can you use the `@extend %identity-placeholder;` here?

Reply:
I couldn't use this directly no. But i have added a version of it that is using ::before pseudo that i am now using. Can be seen in the next commits.